src/main/java/uk/gov/cshr/civilservant/repository/OrganisationalUnitRepository.java

```diff
@@ -32,4 +32,7 @@
 
     @Query(value = "select o.code from organisational_unit o", nativeQuery = true)
     List<String> findAllCodes();
+
+    @Query(value = "select o.code from organisational_unit o where o.id in :organisationIds", nativeQuery = true)
+    List<String> findAllCodesForIds(List<String> organisationIds);
```

Comment:
Method name should be renamed to findAllOrganisationCodesForIds

Reply:
Done. Will be in next commit.